### ISSUE TITLE
question about the curvature loss

### ISSUE THREAD

**Issue by leventt** [RaduAlexandru/permuto_sdf#19]
First of all thanks for the great work and making this code available!

I was going through the code to better understand your work when I noticed the curvature loss isn't clamped for the negative part even though, mean error is calculated on the absolute clamped values here:
https://github.com/RaduAlexandru/permuto_sdf/blob/997dd70892685f8b43e76d58a68a18d2318b1405/permuto_sdf_py/train_permuto_sdf.py#L362

Should it have been clamped between -0.5 and 0.5 instead or in other words should it have been clamped after the absolute values were taken?
(Wouldn't it be more accurate to what was intended, as follows?)
```python
loss_curvature=(torch.clamp(sdf_curvature.abs(), max=0.5).view(-1)).mean()
```

**Comment by RaduAlexandru** [comment]
Hi @leventt, 

Thanks a lot for the interest in the work and for spotting the error. 
I think you're right, the clamping before abs didn't make much sense. In hindsight the whole clamping and abs didn't make a difference since the sdf_curvature was already in range [0.0, 0.5]. 
Therefore it can be safely removed and simplified. I pushed the more simplified version of the curvature loss now.
